mongodb-panache-quickstart/src/main/resources/application.properties

```diff
@@ -1,4 +1,6 @@
 # configure the MongoDB client for a single node cluster on localhost
 quarkus.mongodb.connection-string = mongodb://localhost:27017
 # mandatory if you don't specify the name of the database using @MongoEntity
-quarkus.mongodb.database = person
\ No newline at end of file
+quarkus.mongodb.database = persons
+
+quarkus.log.category."io.quarkus.mongodb.panache.runtime".level=DEBUG
```

Comment:
Please avoid DEBUG level

Reply:
I could remove it ? the default value is INFO no ?

Reply:
Yes, please remove it